src/main/java/org/arl/fjage/param/NamedParameter.java

```diff
@@ -74,20 +74,22 @@ public String toString() {
   }
 
   /**
-   * Compares if two named parameters are equals.
+   * Compares if a parameter is equivalent to the named parameter.
    *
    * @return true if two parameters are equal
    */
   @Override
   public boolean equals(Object obj) {
```

Comment:
This implementation of `equals()` is not an equivalence relation: we have `new NamedParameter("address").equals(NodeInfoParam.address)` but `!NodeInfoParam.address.equals(new NamedParameter("address"))`, so the relation is not symmetric. This may lead to unexpected behaviour e.g. if you stick parameters into a set or map. Instead of overloading `equals()`, we should move this method into a separate function.

Reply:
Introducing a new method will entail special handling of `NamedParameter` in `ParameterRsp`, as general `Parameter` (essentially an `Enum`) will not provide an implementation. In that case, we might as well consider moving the matching logic to `ParameterRsp` and not bother with a separate method.